File: dart/dart-linkedhashmap-contains-key-value/Example2.dart
// https://codevscolor.com/dart-linkedhashmap-contains-key-value

import 'dart:collection';

void main() {
  final linkedHMap = LinkedHashMap.from(
      {1: 'one', 2: 'two', 3: 'three', 4: 'four', 5: 'five', 6: 'six'});

  final value = 'six';

  final isAvailable = linkedHMap.values.any((element) => element == value);

  print('isAvailable: ${isAvailable}');
}

File: kotlin/kotlin-add-items-to-list/example2.kt
// https://codevscolor.com/kotlin-add-items-to-list
fun main() {
    val givenList = mutableListOf("one", "two", "three")
    givenList.addAll(0,arrayListOf("-3", "-2", "-1", "0"))
    
    println(givenList)
}